frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileViewProviderFactory.java

```diff
@@ -50,7 +52,9 @@ private void subscribeToGitRepositoryChanges(Project project, Language language)
           FileContentUtilCore.reparseFiles(psiFile.getVirtualFile());
         }
       });
-      project.getMessageBus().connect().subscribe(topic, listener);
+      MessageBusConnection messageBusConnection = project.getMessageBus().connect();
+      messageBusConnection.subscribe(topic, listener);
+      Disposer.register(project, messageBusConnection);
```

Comment:
Pls enforce this invocation of `Disposer#register` with an ArchUnit test (i.e. all methods that call `MessageBusConnection#subscribe` must also call ` Disposer#register`)